README.md

```diff
@@ -1,4 +1,4 @@
-![](https://github.com/google/pprof/workflows/ci/badge.svg)
+[![](https://github.com/google/pprof/workflows/ci/badge.svg)](https://github.com/google/pprof/actions)
```

Comment:
Any reason you use empty description, unlike the other two links below?

Reply:
I was following https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository.

I originally thought this text will change the text being rendered. But actually it doesn't really matter here.
Now I changed those three names to be more consistent.